inappmessaging/build.gradle

```diff
@@ -10,6 +10,7 @@ apply plugin: 'kotlin-kapt'
 
 android {
   defaultConfig {
+    resValue 'string', 'inappmessaging__version', project.version
```

Comment:
What's this for? The analytics sdk_info object?

Reply:
yes, for IAM to be included in supported modules

Reply:
On iOS we're able to do this solely from the Analytics SDK without requiring the setting of any variables in the SDKs-to-be-tracked.  Is that possible on android?

Reply:
The only other way is to use the package manager, but the Analytics SDK should know all the package name (hard-coded) of the supported modules.

Reply:
Based on additional investigation. Using package manager won't work.
This approach is the only way for now.